### PR TITLE
docs(chart):Add KB for custom legend

### PR DESCRIPTION
**Pull request by NansiYancheva**

### CODE REVIEW

File: knowledge-base/chart-custom-legend.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
+---
+title: Custom Chart Legend
+description: How to design my own chart legend
```
Comment (dimodi) [inline knowledge-base/chart-custom-legend.md:3]:
```suggestion
description: How to create a custom Chart legend that is outside the component and includes rich HTML and additional components.
```

File: knowledge-base/chart-custom-legend.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+type: how-to
+page_title: design custom chart legend diffent from the built-in
```
Comment (dimodi) [inline knowledge-base/chart-custom-legend.md:5]:
```suggestion
page_title: How to Create a Custom Chart Legend
```

File: knowledge-base/chart-custom-legend.md [hunk 3]
```diff
@@ -0,0 +1,207 @@
+---
+title: Custom Chart Legend
+description: How to design my own chart legend
+type: how-to
+page_title: design custom chart legend diffent from the built-in 
+slug: chart-custom-legend
```
Comment (dimodi) [inline knowledge-base/chart-custom-legend.md:6]:
```suggestion
slug: chart-kb-custom-legend
```

File: knowledge-base/chart-custom-legend.md [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+position: 
+tags: telerik, blazor, chart, legend, css, html
+ticketid: 1637556, 1640086
+res_type: kb
+---
+
+## Environment
```
Comment (dimodi) [inline knowledge-base/chart-custom-legend.md:13]:
```suggestion
## Environment

```

File: knowledge-base/chart-custom-legend.md [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+<table>
```
Comment (dimodi) [inline knowledge-base/chart-custom-legend.md:14]: Use 4 spaces instead of tabs

File: knowledge-base/chart-custom-legend.md [hunk 6]
```diff
@@ -0,0 +1,207 @@
...
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Chart for Blazor</td>
+		</tr>
+	</tbody>
+</table>
+
+## Description
```
Comment (dimodi) [inline knowledge-base/chart-custom-legend.md:23]:
```suggestion
## Description

```

File: knowledge-base/chart-custom-legend.md [hunk 7]
```diff
@@ -0,0 +1,207 @@
...
+How to change the Chart Legend graphics and apply my own design? I want to have different Chart Legend Title and Icons. How to customize the Chart Legend UI.
```
Comment (dimodi) [inline knowledge-base/chart-custom-legend.md:24]:
```suggestion
This knowledge base article answers the following questions:

* How to customize the Chart legend UI?
* How to change the Chart legend graphics? I want to use custom markers and define the color of the legend item labels.
```

File: knowledge-base/chart-custom-legend.md [hunk 8]
```diff
@@ -0,0 +1,207 @@
...
+
+## Solution
```
Comment (dimodi) [inline knowledge-base/chart-custom-legend.md:26]:
```suggestion
## Solution

```

File: knowledge-base/chart-custom-legend.md [hunk 9]
```diff
@@ -0,0 +1,207 @@
...
+Change the Chart Legend design with HTML/CSS. You need to:
```
Comment (dimodi) [inline knowledge-base/chart-custom-legend.md:27]:
```suggestion
To use a custom Chart legend, you need to:

```

File: knowledge-base/chart-custom-legend.md [hunk 10]
```diff
@@ -0,0 +1,207 @@
...
+* Disable the built-in legend
+* Use HTML markup or any other components to create the custom legend content
```
Comment (dimodi) [inline knowledge-base/chart-custom-legend.md:29]:
```suggestion
1. Disable the built-in legend with `<ChartLegend Visible="false" />`
1. Define specific colors for the Chart series or data points [through `Color` or `ColorField`]({%slug components/chart/types/column%}).
1. Implement the custom legend UI outside the Chart.
1. Use the same Chart series or data point colors in custom Chart legend.
```

File: knowledge-base/chart-custom-legend.md [hunk 11]
```diff
@@ -0,0 +1,207 @@
...
+
+>caption Custom Chart Legend with ListView
+
+````CSHTML
+@*Chart with custom legend*@
+
```
Comment (dimodi) [inline knowledge-base/chart-custom-legend.md:35]:
```suggestion
```

File: knowledge-base/chart-custom-legend.md [hunk 12]
```diff
@@ -0,0 +1,207 @@
...
+## Description
+How to change the Chart Legend graphics and apply my own design? I want to have different Chart Legend Title and Icons. How to customize the Chart Legend UI.
+
+## Solution
+Change the Chart Legend design with HTML/CSS. You need to:
+* Disable the built-in legend
+* Use HTML markup or any other components to create the custom legend content
+
+>caption Custom Chart Legend with ListView
```
Comment (dimodi) [inline knowledge-base/chart-custom-legend.md:31]:
```suggestion
>caption Custom Chart Legend with a ListView
```

File: knowledge-base/chart-custom-legend.md [hunk 13]
```diff
@@ -0,0 +1,225 @@
...
+        public double Value { get; set; }
+        public string Color { get; set; } = string.Empty;
+        public int Year { get; set; }
+    }
+}
+````
+
+## See Also
+
+[Override Theme Styles]({%slug themes-override%})
```
Comment (dimodi) [inline knowledge-base/chart-custom-legend.md:225]:
```suggestion
* [Chart Legend Documentation]({%slug chart-legend%})
* [Chart Legend Customization Demo](https://demos.telerik.com/blazor-ui/chart/legend-customization)
```

**Review by dimodi — APPROVED** [review]

File: knowledge-base/chart-custom-legend.md [hunk 14]
```diff
@@ -0,0 +1,233 @@
...
+This knowledge base article answers the following questions:
+
+* How to customize the Chart legend UI?
+* How to change the Chart legend graphics? I want to use custom markers and define the color of the legend item labels.
+
+## Solution
+
+To use a custom Chart legend, you need to:
+
+1. Disable the built-in legend with `<ChartLegend Visible="false" />`
```
Comment (ikoevska) [inline knowledge-base/chart-custom-legend.md:35]: You need some ending punctuation here.

File: knowledge-base/chart-custom-legend.md [hunk 15]
```diff
@@ -0,0 +1,233 @@
...
+1. Define specific colors for the Chart series or data points [through `Color` or `ColorField`]({%slug components/chart/types/column%}).
+1. Implement the custom legend UI outside the Chart.
+1. Use the same Chart series or data point colors in custom Chart legend.
```
Comment (ikoevska) [inline knowledge-base/chart-custom-legend.md:38]:
```suggestion
1. Use the same Chart series or data point colors in the custom Chart legend.
```

**Review by ikoevska — COMMENTED** [review]
Reviewed.

**Pull request merged**